FuelSDK-CSharp/FuelReturn.cs

```diff
@@ -479,12 +479,12 @@ protected string ExecuteFuel(FuelObject obj, string[] required, string method, b
 				foreach (string urlProp in obj.URLProperties)
 					completeURL = completeURL.Replace("{" + urlProp + "}", string.Empty);
 
-			completeURL += "?access_token=" + obj.AuthStub.AuthToken;
-			if (obj.Page != 0)
-				completeURL += "&page=" + obj.Page.ToString();
+            if (obj.Page != 0)
+                completeURL += "?page=" + obj.Page.ToString();
 
-			var request = (HttpWebRequest)WebRequest.Create(completeURL.Trim());
-			request.Method = method;
+            var request = (HttpWebRequest)WebRequest.Create(completeURL.Trim());
+            request.Headers.Add("Authorization", "Bearer " + obj.AuthStub.AuthToken);
+            request.Method = method;
 			request.ContentType = "application/json";
 			request.UserAgent = ETClient.SDKVersion;
```

Comment:
@sfdrogojan the value of ETClient.SDKVersion says FueSDX.C#. Need to confirm with Eric if thats right or it should say FuelSD**K**

Reply:
public const string SDKVersion = "FuelSDX-C#-v1.0.0"; I can change this if needed.